string self_ip
WiFiNeighbour[] neighbours
